### PR TITLE
Pygame cheat sheet

### PR DESCRIPTION
**Pull request by amarlearning**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Pygame Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
What changes does this PR introduce? 
New Goodie: Python Cheat Sheet

###### Related Issues or Discussions 
What issues or discussions (if any) does this PR address? Please provide the issue # or link.

###### People to notify
@mention any relevant people/organizations
@pjhampton 

---

Instant Answer Page:  https://duck.co/ia/view/pygame_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@amarlearning Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pygame cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pygame_cheat_sheet_mobile.png?G08xtxbIrs)

**Comment by tagawa** [comment]
Thanks @amarlearning! This is now on the beta server for testing: https://beta.duckduckgo.com/?q=pygame+cheat+sheet&ia=cheatsheetscontroller&iax=1

File: share/goodie/cheat_sheets/json/pygame.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
+{
+    "id": "pygame_cheat_sheet",
+    "name": "pygame",
+    "description": "A reference on the Pygame 2D game framework",
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/pygame.json]: _to the_ not _on the_

**Comment by pjhampton** [comment]
Apart from that one issue, this looks nice

Thanks @amarlearning 💙  💛  ❤️  💚

File: share/goodie/cheat_sheets/json/pygame.json (hunk 1 above)
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/pygame.json]: @pjhampton corrected!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pygame cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pygame_cheat_sheet_mobile.png?PAmVwXkinT)

File: share/goodie/cheat_sheets/json/pygame.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        "Time",
+        "Events",
+        "Fonts"
+    ],
+
+    "sections": {
+        "Load and Launch Pygame" : [
+            {
+                "key": "import pygame",
+                "val": "The 'import pygame' imports the package with all the available Pygame modules."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/pygame.json]: You can remove `the 'import pygame'` and just start the sentence with `Imports the package...`, similar to the rest of the cheatsheet.

File: share/goodie/cheat_sheets/json/pygame.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                "key": "pygame.display.update()",
+                "val": "Redraws the main display surface if argument list is empty."
+            },
+            {
+                "key": "pygame.display.get_surface()",
+                "val": "Returns a reference to the Surface instantiated with the set_mode() function."
+            },
+            {
+                "key": "screen = pygame.display.set_mode((width, height))",
+                "val": "Initializes and creates the window where your game will run.'"
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/pygame.json]: @tagawa thanks for pointing out. corrected!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pygame cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pygame_cheat_sheet_mobile.png?MsfEfN5up8)

**Comment by tagawa** [comment]
Thank you @amarlearning, and @pjhampton for the review. Let's merge! This should go live in a few days.

**Comment by daxtheduck** [comment]
@amarlearning this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**